components/04-indicators/icon/icon.hbs

```diff
@@ -1 +1,3 @@
-<span class="{{ icon }}" aria-hidden="true"></span>
\ No newline at end of file
+<svg class="di di-{{ icon }}">
+  <use href="{{path '/dso-icons.svg' }}#{{ first (split icon ' ') }}" />
+</svg>
```

Comment:
Ik snap een beetje wat het doet (of meer: wat het gewenste eindresultaat moet zijn), maar waarom is het nodig? Of ga je erva dat de string altijd 'icon<spatie>times' is?

Reply:
`icon` kan `user` of `times modifier1 modifier2` kan zijn. Ik heb het gemaakt voor de carousel: Bootstrap verwacht dat de volgende/vorige knoppen OF een `glyphicon` hebben (wat we nu niet meer doen) OF `icon-prev` of `icon-next`.

`icon-prev` en `icon-next` zijn modifiers. Het icoon + modifiers moet op het root `svg` element. maar _alleen_ het icoon moet in de `use href` gebruikt worden